plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -378,10 +381,21 @@ protected static String formatImageName(@Nonnull String repository, @Nonnull Str
 
   @Nonnull
   protected DockerClient openDockerClient() throws MojoExecutionException {
+    ContainerRegistryAuthSupplier authSupplier = null;
+    try {
+      authSupplier = ContainerRegistryAuthSupplier.forApplicationDefaultCredentials()
+          .build();
+      getLog().info("Using Google application credentials");
+    } catch (IOException ex) {
```

Comment:
This needs to catch `DockerException` as well, which maybe shouldn't be an unchecked exception to begin with? (I realize that would be a breaking change)